src/codemodder/codemods/jwt_decode_verify.py

```diff
@@ -0,0 +1,86 @@
+import libcst as cst
+from libcst import matchers
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import SemgrepCodemod
+
+
+class JwtDecodeVerify(SemgrepCodemod):
+    NAME = "jwt-decode-verify"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Makes any of the multiple `verify` parameters to a `jwt.decode` call be `True`."
+
+    @classmethod
+    def rule(cls):
+        return r"""
+            rules:
+                - pattern-either:
+                  - patterns:
+                      - pattern: jwt.decode(..., verify=False, ...)
+                      - pattern-inside: |
+                          import jwt
+                          ...
+                  - patterns:
+                      - pattern-regex:  |-
+                            jwt.decode\(.*options={.*"verify_(signature|exp|nbf|iat|aud|iss)": False.*}.*\)
```

Comment:
There are a lot of problem by just using a plain regex here (whitespaces and quotes, to name a couple). 

I'd suggest following semgrep doc's example on how to match dict keys:
https://semgrep.dev/docs/writing-rules/pattern-examples/

Reply:
I don't think this is half bad as it looks, given that we rely on matching that exact pattern or not matching at all. But good find! Going to see if I can adapt it.